src/pybind/mgr/dashboard/frontend/src/app/shared/components/pwd-expiration-notification/pwd-expiration-notification.component.ts

```diff
@@ -30,8 +30,9 @@ export class PwdExpirationNotificationComponent implements OnInit {
         } else {
           this.alertType = 'warning';
         }
-
-        this.authStorageService.isPwdDisplayedSource.next(true);
+        this.displayNotification =
```

Comment:
This was introduced in https://github.com/ceph/ceph/pull/35292.
We need to backport that PR before we merge this one.

Reply:
backport is already created: https://github.com/ceph/ceph/pull/35772